### ISSUE TITLE
Two issues. Maybe just chef Windows support. End of chocolatey install throws exception

### ISSUE THREAD

**Issue by lonniev** [chocolatey/chocolatey-cookbook#37]
Starting with chef 12.3.0, recipes using the chocolatey resource are throwing an exception as they successfully conclude. Here's a bit of the log. These both appear to be related to Chef having problems regex processing pathnames that have Windows path separators and regex trying to handle those as special characters.

(The installs are completing successfully because I can visit the guest and see that the requested package was indeed installed even though the chef run ended with the exception.
 
```bash
==> vb-tt-sync: [2015-05-12T12:43:37-07:00] DEBUG: providers that support resource execute[install package google-chrome-x64] include: [Chef::Provider::Execute]
==> vb-tt-sync: [2015-05-12T12:43:37-07:00] DEBUG: providers that survived replacement include: [Chef::Provider::Execute]
==> vb-tt-sync: C:/opscode/chef/embedded/apps/chef/lib/chef/formatters/error_inspectors/resource_failure_inspector.rb:66: warning: invalid Unicode Property \p
==> vb-tt-sync: 
==> vb-tt-sync: [2015-05-12T12:44:33-07:00] INFO: Running queued delayed notifications before re-raising exception
==> vb-tt-sync: [2015-05-12T12:44:33-07:00] INFO: Running queued delayed notifications before re-raising exception
==> vb-tt-sync: [2015-05-12T12:44:33-07:00] DEBUG: Re-raising exception: RegexpError - invalid backref number/name: /^C:\tmp\vagrant-chef\6c0f26688ebda1accbe1d0ab76a99d3f\cookbooks\chocolatey-chrome\recipes\default.rb:([\d]+)/
```

**Comment by ferventcoder** [comment]
@mwrock @guilhem Can you verify?

**Comment by ferventcoder** [comment]
unicode property issue is what it looks like.

**Comment by ferventcoder** [comment]
why I don't know though.

**Comment by lonniev** [comment]
Just guessing here and looking for where a "\P" might occur in those choco installs. I see that the command line passed to Execute is "C:\ProgramData\chocolatey..." and that starts right off with the necessary victim "\P".

I also see that the usages of `chocolatey pkg do... action :upgrade` do NOT suffer the same exception.

Therefore, is recent chocolatey returning a new error code for success that the chocolatey cookbook recipe and its surrounding chef is not prepared to recognize as success?

choco version reports 0.9.8.28

(I can retry after asking the recipes to update choco itself first.)

**Comment by lonniev** [comment]
This issue https://github.com/chef/chef-dk/issues/389 is related. Different unicode char (\u rather than \p) but the stumble occurs in the same chef error handling code.

Therefore:

1. why is choco claiming an error?
1. when will chef modify its code for dealing with Windows paths?

If I recall, (2) is being resolved already.

**Comment by lamont-granquist** [comment]
Probably fixed by chef/chef#3357

**Comment by lonniev** [comment]
I agree with @lamont-granquist
